v1/features.md

```diff
@@ -198,17 +198,14 @@ $this->arParams['AJAX_HEAD_RELOAD'] = 'Y';
 
 ```javascript
 // script.js
-$(function() {
-	$(document).on('click', '.js-ajax', function() {
-		$.ajax({
-			type: 'GET',
-			data: $(this).data('ajax-params'),
-			success: function (response) {
-				$('#ajax-block').html(response);
-			}
-		});
+$(document).on('click', '.js-ajax', function() {
+	$.ajax({
+		data: $(this).data('ajaxParams'),
```

Comment:
да, всё верно, так рекомендует документация по jquery https://api.jquery.com/data/#data-html5
> The second statement of the code above correctly refers to the data-last-value attribute of the element. In case no data is stored with the passed key, jQuery searches among the attributes of the element, converting a camel-cased string into a dashed string and then prepending data- to the result. So, the string lastValue is converted to data-last-value.

К тому же такая нотация упрощает понимание кода, когда необходимо обратиться к атрибуту.
Например: 
```javascript
// <div class="element" data-first-prop="1" data-second-prop="2"></div>
var $data = $('.element').data();
console.log($data.firstProp); // $data.first-prop не сработает
```